### ISSUE TITLE
why default epoch = 200?

### ISSUE THREAD

**Issue by Skyorca** [junyanz/pytorch-CycleGAN-and-pix2pix#489]
I have no idea why default epoch reaches 200, but I see iter=100. Should I reduce #iter to reduce epoch, or just name it using comman line like --epoch 100, etc?

**Comment by shreyasbapat** [comment]
That means --epoch is a parameter?

**Comment by junyanz** [comment]
`--epoch` is not. We have `--niter` and `--niter_decay`. If the learning rate policy `linear` is used, the number of epochs = `--niter` and `--niter_decay`.  See [here](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/models/networks.py#L31) for more details. Also check out the training option [class](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/options/train_options.py).

**Comment by SunHaozhe** [comment]
So why did you fix the default opt.niter=100 and opt.niter_decay=100 ? 

I mean, did you find out that this setting is suitable for the majority of tasks or you just set these numbers randomly ? In your opinion, should I pay attention to change this setting when I train my own dataset ?

**Comment by junyanz** [comment]
It works for most of the applications reported in the paper. The parameters depend on your task and the number of training images. Let's say you have 1 million images, you probably only need a few epochs. I recommend that you save checkpoints frequently and pick up the best one after training.
